core/domain/rte_component_registry_test.py

```diff
@@ -119,9 +119,10 @@ def test_default_rte_components_are_valid(self):
             self.assertTrue(os.path.isdir(component_dir))
 
             # In this directory there should be a config .py file, an
-            # html file, a JS file, a .png file and a protractor.js file.
+            # html file, a JS file, an icon .png file and a protractor.js file,
+            # and an optional preview .png file.
             dir_contents = self._listdir_omit_ignored(component_dir)
-            self.assertLessEqual(len(dir_contents), 5)
+            self.assertLessEqual(len(dir_contents), 6)
```

Comment:
Should probably add some additional checks in the lines below to enforce naming conventions.